Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -131,6 +132,8 @@ void EnhancementsWidget::ConnectWidgets()
           [this](int) { SaveSettings(); });
   connect(m_3d_mode, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged),
           [this](int) { SaveSettings(); });
+  connect(m_configure_pp_effect, static_cast<void (QPushButton::*)()>(&QPushButton::released),
```

Comment:
This cast should be unnecessary, I think? (The other casts are necessary because `currentIndexChanged()` is overloaded.)